DEV-8/src/SizeGetterFromConsole.java

```diff
@@ -0,0 +1,70 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+
+/**
+ * Uses to get dimensions of matrixs from console
+ * Contains validating
+ */
+public class SizeGetterFromConsole {
+
+  /**
+   * ****************************************************
+   * e.g. we have array dimen [x,y]                     *
+   *                                                    *
+   * the 0th element of returning array is x            *
+   * the 1st element of returning array is y            *
+   ******************************************************
+   *
+   * also validating inputted variables in other method
+   *
+   * @return array with ints
+   */
+  public int[] getSizeOfMatrix(){
```

Comment:
Method with name getSize... can't returns massive
Better to split to two methods- get X and Y dimensions separately